spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.